test/environment_test.rb

```diff
@@ -23,17 +23,30 @@
         Dir.chdir($pwd)
       end
 
-      it 'sets env vars from .env' do
-        ENV['FOO'].must_equal 'bar'
-      end
+      describe 'ignores .env' do
+        it 'does not sets env vars' do
+          ENV['FOO'].must_be_nil
+        end
 
-      it 'sets port from .env' do
-        @env.port.must_equal 42
+        it 'does not sets port' do
+          @env.port.must_equal 2300
```

Comment:
@aderyabin Please set the port in a `.env.test` fixture and verify that this still works. Please have a look at https://github.com/hanami/hanami/pull/392